### PR TITLE
defining error and getClass by year

### PR DESCRIPTION
**Pull request by sudddy**

### CODE REVIEW

**Comment by theappsolutes-vinoth** [comment]
What about using different error codes for each messages ?
It will also help us in displaying messages in warning, danger and success alert boxes. Lets say code 04 for invalid json - warning, code 01 for invalid parameters - danger.
What do you think ? @sudddy  @aish1007

**Comment by sudddy** [comment]
> What about using different error codes for each messages ?
> It will also help us in displaying messages in warning, danger and success alert boxes. Lets say code 04 for invalid json - warning, code 01 for invalid parameters - danger.
> What do you think ? @sudddy @aish1007

Yeah, that would be great. I'll do it.

File: routes/class/class.js [hunk 1]
```diff
@@ -1,36 +1,81 @@
...
+  if (!year) {
+    error.errorMsg = errorMessages.IMPROPER_REQUEST;
+    return res.send(error);
+  }
 
-      console.log(result);
+  try {
+    ClassInfo.find({ year: year }, function(err, result) {
+      if (err) {
+        throw err;
```
Comment (AishwaryaN10) [inline routes/class/class.js:32]: We should send the response back to user instead of throwing it.

Comment (aish1007) [inline routes/class/class.js:32]: We should send the response back to the user instead of throwing it.

**Pull request merged**